webpack.common.js

```diff
@@ -1,160 +0,0 @@
-/* global __dirname module */
```

Comment:
If possible, could you revert this and re-do the commit using `git mv` so that it registers as a rename/move action? Since git thinks it's a delete/add, we will lose commit history for this file.

Reply:
I can open a new PR?

Reply:
okay

Reply:
I tried using git mv on a fresh branch, but it resulted in the behaviour it currently shows..

Reply:
You might have to do the `git mv` of the original files to their new locations, commit that, and then make the changes and commit those. It might detect it as a remove/add if the file is not 1:1 across locations in the commit itself.

Reply:
Alright, I'll do that!

Reply:
Done. If you follow the commits done in this PR, you'll see that it automatically converts a move + modify into creating a file, so 🤷🏾

Reply:
@evenstensberg Yep! You've done everything correctly here, so I'm chalking it up to github weirdness at this point? Pretty baffling. Thanks for doing that.

Reply:
Yea.. Lemme rebase and make sure that tests are green before this is ready to review again.

Reply:
Rebased now. Some ci tasks are failing though, is there something you are aware of? I can open a new PR that's more compact than this one. Lots of rebase and force pushes

Reply:
No worries about the PR, this should be fine. The failing test is a long-standing flaky test, I'll give it a re-run and we should be able to get a green build. Once that's done, I'm just going to confirm we still have your CLA on file with @jvigliotta and this should be good to go.